Sources/Public/Animation/LottieView.swift

```diff
@@ -380,4 +400,16 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  private func clearAnimationIfNecessary() {
+    if loadAnimation != nil, showPlaceholderWhileReloading {
+      animationSource = nil
+    }
+  }
+}
+
+// MARK: - DefaultLoadTrigger
+
+/// The default load animation trigger token
+private enum DefaultLoadTrigger: Hashable {
```

Comment:
Why is this necessary now if we didn't have it before?

Reply:
I removed the `Optional` in the `fetchTrigger` binding so that the code could compile with the `any Equatable` changes. We use this as the default value for the binding now. Hopefully we can remove this if we remove the `any Equatable` changes.

Reply:
I was able to remove this